jobclient/python/cook/scheduler/client/instance.py

```diff
@@ -201,8 +184,11 @@ def to_dict(self) -> dict:
     def from_dict(cls, d: dict) -> 'Instance':
         """Create an instance from its `dict` representation."""
         d = deepcopy(d)
-        d['uuid'] = UUID(d['uuid'])
+        d['task_id'] = UUID(d['task_id'])
+        d['start_time'] = datetime.fromtimestamp(d['start_time'] / 1000)
         d['status'] = Status.from_string(d['status'])
+        if 'end_time' in d:
+            d['end_time'] = datetime.fromtimestamp(d['end_time'] / 1000)
```

Comment:
Since we do this in a couple of spots, should we have a util function `milliseconds_to_datetime` or similar?